### PR TITLE
Add Meeting Minutes for Labs Meeting on 2015-12-11

### PR DESCRIPTION
**Pull request by JevinSidhu**
Hey @zachlatta, is this fine?

Let me know if anything needs tweaking before we merge :smile:

### CODE REVIEW

File: meetings/2015-12-11_labs_1.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# Labs Meeting - 2015-12-11
+
+*Present : Jevin, Vaibhav, Harrison, Lelia, Erina, Jasmine, Chaoyi (Troy), Selynna*
+
+#### Cipher Problem: Getting people's attention outside of club time
```
Comment (MaxWofford) [inline meetings/2015-12-11_labs_1.md]: Make sure to capitalize titles!

File: meetings/2015-12-11_labs_1.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
+# Labs Meeting - 2015-12-11
+
+*Present : Jevin, Vaibhav, Harrison, Lelia, Erina, Jasmine, Chaoyi (Troy), Selynna*
```
Comment (paked) [inline meetings/2015-12-11_labs_1.md]: Chaoyi is Chaoyi

File: meetings/2015-12-11_labs_1.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+#### Cipher Problem: Getting people's attention outside of club time
+  - Erina + Lelia: Clubs are inside of school so usually we just ask them during school
+    - Talk to them in-person  + help through Cloud9's online functionality
+  - During the club: sign up for Slack + Remind
+  - Harrison: Messenger chat for Facebook
+    - 15 people in the club
+    - Works effectively since everyone is on Facebook usually at least a couple times a day
+
+#### Cloud9 versus JS Bin
```
Comment (MaxWofford) [inline meetings/2015-12-11_labs_1.md]: Headers should have Newlines before and after

Comment (JevinSidhu) [inline meetings/2015-12-11_labs_1.md]: Added! Thanks for the reminder :)

File: meetings/2015-12-11_labs_1.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+- Harrison: Flows well, but it is not written clearly enough
+  - Some parts are dodgy (I see what you did there, Harrison -Jevin)
+  - Feels kinda a draft
+  - Definitely needs some work
+
+#### Structuring of the Workshops
+
+- Jevin: Doing something harder, then easier in a workshop
+  - Having the feeling of, "Okay, I kind of get this," and then going into the excitement of knowing something 100% is key
```
Comment (MaxWofford) [inline meetings/2015-12-11_labs_1.md]: Lines should be wrapped at 80 characters.

**Comment by JevinSidhu** [comment]
@MaxWofford We made it! Sorry for all my mistakes with the formatting haha, but I think next time I should have it down.

**Comment by zachlatta** [comment]
Absolutely no worries re: formatting! It's super difficult to get started with and we all learn with time :-).

**Comment by MaxWofford** [comment]
@JevinSidhu Nice work! The tests are passing. The last step is to rebase it so we don't have extra commits in the pull request.

**Comment by JevinSidhu** [comment]
Yay! Thanks @zachlatta :100:

**Comment by MaxWofford** [comment]
Nice work @JevinSidhu! Thanks for adding the notes :)

**Pull request merged**